src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.UserResponseDto;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.UserDtoMapper;
+import mate.academy.spring.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserDtoMapper userDtoMapper;
+
+    @Autowired
+    public UserController(UserService userService, UserDtoMapper userMapper) {
+        this.userService = userService;
+        this.userDtoMapper = userMapper;
+    }
+
+    @GetMapping
+    public List<UserResponseDto> getAll() {
+        return userService.listUsers().stream()
+                .map(userDtoMapper::parse)
+                .collect(Collectors.toList());
+    }
+
+    @GetMapping("/{userId}")
+    public UserResponseDto get(@PathVariable Long userId) {
+        return userDtoMapper.parse(userService.getById(userId));
```

Comment:
```suggestion
        return userDtoMapper.parseToDto(userService.getById(userId));
```
we'll have response and request DTOs, so it's better to specify which way you map the object